client/post-editor/editor-html-toolbar/style.scss

```diff
@@ -190,9 +185,9 @@
 			color: $gray;
 		}
 		span {
-			color: darken( $gray, 30% );
+			color: $gray-text-min;
```

Comment:
It's very minor, but to be consistent with other labels, I think the text colour here should be `darken( $gray, 20% );` which is `#4f748e`.

Reply:
This is actually something I meant to point out and I forgot.
Recently [we talked](https://github.com/Automattic/wp-calypso/pull/18624#issuecomment-335421073) about text colors with @folletto and we settled on using `$gray-text-min` to simplify future tweaks.

In the end the difference with `darken( $gray, 20% )` isn't really visible (at least to my coder eyes 😄 ), so I'm not really opposed on using `darken` instead.

Reply:
I wouldn't have realised if I didn't see the diff :) Ignore me then, thanks for the info!